src/components/ADempiere/Form/VPOS/Order/orderLineMixin.js

```diff
@@ -186,6 +200,53 @@ export default {
           })
         })
     },
+    formatDate2(date) {
```

Comment:
What is it? already exist a util class for format date and the name must be more explicit